### ISSUE TITLE
TripDescriptor.direction_id is not supported

### ISSUE THREAD

**Issue by barbeau** [kurtraschke/gtfs-rt-dump#2]
When exporting a GTFS-realtime .pb file, I'm seeing TripDescriptor.direction_id get exported like this:

~~~
    trip {
      trip_id: "15837958-DEC19-RAIL-Weekday-01"
      schedule_relationship: SCHEDULED
      route_id: "901"
      6: 1
    }
~~~

So the project just needs to be updated to the latest gtfs-realtime.proto.

@kurtraschke What are your thoughts on switching from the OBA GTFS-RT exportor to [gtfs-realtime-bindings](https://github.com/MobilityData/gtfs-realtime-bindings)? Or are you using extensions that would prevent this?

**Comment by kurtraschke** [comment]
Yes, I'm using [onebusaway-gtfs-realtime-api](https://github.com/OneBusAway/onebusaway-gtfs-realtime-api) for the extensions; I've opened onebusaway/onebusaway-gtfs-realtime-api#16 to bump the upstream dependency.
